### PR TITLE
Bump @nteract/ packages and flow

### LINKED ISSUE #1169: An in-range update of @nteract/transforms is breaking the build 🚨
## Version **3.1.1** of [@nteract/transforms](https://www.npmjs.com/package/@nteract/transforms) was just published.

<table>
  <tr>
    <th align=left>
      Branch
    </th>
    <td>
      <a href="/nteract/hydrogen/compare/greenkeeper%2F%40nteract%2Ftransforms-3.1.1">Build failing 🚨</a>
    </td>
  </tr>
  <tr>
    <th align=left>
      Dependency
    </td>
    <td>
      @nteract/transforms
    </td>
  </tr>
  <tr>
    <th align=left>
      Current Version
    </td>
    <td>
      3.0.3
    </td>
  </tr>
  <tr>
    <th align=left>
      Type
    </td>
    <td>
      dependency
    </td>
  </tr>
</table>

This version is **covered** by your **current version range** and after updating it in your project **the build failed**.

@nteract/transforms is a direct dependency of this project, and **it is very likely causing it to break**. If other packages depend on yours, this update is probably also breaking those in turn.

<details>
<summary>Status Details</summary>

- ❌ **continuous-integration/travis-ci/push** The Travis CI build could not complete due to an error [Details](https://travis-ci.org/nteract/hydrogen/builds/323646742?utm_source=github_status&utm_medium=notification)
</details>

---

<details>
<summary>FAQ and help</summary>

There is a collection of [frequently asked questions](https://greenkeeper.io/faq.html). If those don’t help, you can always [ask the humans behind Greenkeeper](https://github.com/greenkeeperio/greenkeeper/issues/new).
</details>

---

Your [Greenkeeper](https://greenkeeper.io) Bot :palm_tree:

### LINKED ISSUE #1170: An in-range update of @nteract/transform-plotly is breaking the build 🚨
## Version **3.1.3** of [@nteract/transform-plotly](https://www.npmjs.com/package/@nteract/transform-plotly) was just published.

<table>
  <tr>
    <th align=left>
      Branch
    </th>
    <td>
      <a href="/nteract/hydrogen/compare/greenkeeper%2F%40nteract%2Ftransform-plotly-3.1.3">Build failing 🚨</a>
    </td>
  </tr>
  <tr>
    <th align=left>
      Dependency
    </td>
    <td>
      @nteract/transform-plotly
    </td>
  </tr>
  <tr>
    <th align=left>
      Current Version
    </td>
    <td>
      3.1.2
    </td>
  </tr>
  <tr>
    <th align=left>
      Type
    </td>
    <td>
      dependency
    </td>
  </tr>
</table>

This version is **covered** by your **current version range** and after updating it in your project **the build failed**.

@nteract/transform-plotly is a direct dependency of this project, and **it is very likely causing it to break**. If other packages depend on yours, this update is probably also breaking those in turn.

<details>
<summary>Status Details</summary>

- ❌ **continuous-integration/travis-ci/push** The Travis CI build could not complete due to an error [Details](https://travis-ci.org/nteract/hydrogen/builds/323665518?utm_source=github_status&utm_medium=notification)
</details>

---

<details>
<summary>FAQ and help</summary>

There is a collection of [frequently asked questions](https://greenkeeper.io/faq.html). If those don’t help, you can always [ask the humans behind Greenkeeper](https://github.com/greenkeeperio/greenkeeper/issues/new).
</details>

---

Your [Greenkeeper](https://greenkeeper.io) Bot :palm_tree:

### PR DESCRIPTION
**Pull request by lgeiger**
Closes #1172
Closes #1171
Closes #1168
Closes #1170
Closes #1169 
And finally closes #1044 🎉

### CODE REVIEW

File: lib/components/inspector.js [hunk 1]
```diff
@@ -34,6 +33,7 @@ const Inspector = observer(({ store: { kernel } }: Props) => {
         fontSize: atom.config.get(`Hydrogen.outputAreaFontSize`) || "inherit"
       }}
     >
+      {/* $FlowFixMe React element `Transform`. Expected React component instead of Transform*/}
```
Comment (lgeiger) [inline lib/components/inspector.js]: Don't know what's happening here.

File: .flowconfig [hunk 2]
```diff
@@ -8,5 +8,5 @@ types/
 .*/test/*
 
 [options]
-unsafe.enable_getters_and_setters=true
 esproposal.decorators=ignore
+include_warnings=true
```
Comment (rgbkrk) [inline .flowconfig]: 🆒

File: flow-typed/npm/@jupyterlab/services_vx.x.x.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-// flow-typed signature: a318a3580f3ad3485152ab57c3e413cd
-// flow-typed version: <<STUB>>/@jupyterlab/services_v^0.50.2/flow_v0.54.0
+// flow-typed signature: 3cf3f1e6d0367ca9ce6f1cdbea6ee45b
+// flow-typed version: <<STUB>>/@jupyterlab/services_v^0.52.0/flow_v0.62.0
```
Comment (rgbkrk) [inline flow-typed/npm/@jupyterlab/services_vx.x.x.js]: Ohhhhhh

**Comment by rgbkrk** [comment]
beta channel is failing, is that ok for the time being?f

**Comment by lgeiger** [comment]
> beta channel is failing, is that ok for the time being?

Should be fine, Travis has some problems: https://www.traviscistatus.com/

**Comment by rgbkrk** [comment]
Whoa, I wonder if that's what's been slowing down the infra.

I'll go ahead and merge this. Thanks @lgeiger

**Comment by rgbkrk** [comment]
Onto greater things -- we can now actually stay up to date with flow and ship on the regular, syncing with all the other upstream projects. 😄

**Pull request merged**